src/components/entry/EntryNav.js

```diff
@@ -1,7 +1,70 @@
-const EntryNav = () => (
-  <>
-    <p>EntryNav</p>
-  </>
-)
+import { ArrowBack } from '@styled-icons/boxicons-regular'
+import { Pencil } from '@styled-icons/boxicons-solid'
+import { MapMarkerAlt as Map } from '@styled-icons/fa-solid'
+import styled from 'styled-components'
+
+import { theme } from '../ui/GlobalStyle'
+import IconButton from '../ui/IconButton'
+
+const EntryNavContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const EntryNavTextContainer = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+`
+
+const EntryNavText = styled.p`
+  margin-left: 10px;
+`
```

Comment:
Maybe remove this styled component and put the margin on the svg. With minor/less semantic styling in styled components, we want to avoid creating new containers by using selectors on the main component instead.

Reply:
So in this case in EntryNavTextContainer